Deployment/Jenkins/parent-autoscale.groovy

```diff
@@ -97,108 +97,88 @@ pipeline {
                 git branch: "${GitProjBranch}",
                     credentialsId: "${GitCred}",
                     url: "${GitProjUrl}"
-                writeFile file: 'InfraStack.parms.json',
+                writeFile file: 'sonar.auto.parms.json',
```

Comment:
Would this maybe be better as `sonar-_parent-autoscale.parms.json`?

Reply:
Yes